liquidity/fees.go

```diff
@@ -370,6 +380,8 @@ func (f *FeePortion) loopOutFees(amount btcutil.Amount,
 	// amounts provided by the quote to get the total available for
 	// off-chain fees.
 	feeLimit := ppmToSat(amount, f.PartsPerMillion)
+
+	// Apply the small miner multiplier for the fee budget calculations.
```

Comment:
I think it'd be cleaner if `loopOutFees` would receive a `minerFeeMultiplier` and simply scale by that. This way we wouldn't need to divide by the "max multiplier" when checking against our limits.

Reply:
Yes, but in that solution other structs like the loop builder would have to know when/how to use the miner multipliers.